webapp/src/components/containers/ButtonsCard.vue

```diff
@@ -0,0 +1,54 @@
+<template>
+  <div
+    class="flex flex-col p-7 gap-4 bg-white rounded-lg transition duration-200 group hover:-translate-y-1 hover:scale-[101%] hover:outline hover:outline-2 hover:outline-disco-cyan hover:cursor-pointer"
+  >
+    <div
+      class="text-xl text-disco-blue group-hover:text-disco-cyan"
+      :class="`text-${titleAlign}`"
+    >
+      <slot name="title" />
+    </div>
+
+    <div class="text-slate-500">
+      <slot />
+    </div>
+
+    <div
+      class="flex flex-wrap mt-auto items-center"
+      :class="`justify-${buttonsJustify}`"
```

Comment:
```suggestion
      class="flex flex-wrap mt-auto items-center gap-x-4"
      :class="`justify-${buttonsJustify}`"
```
Suggested change:
![Screenshot 2024-07-01 at 15 16 23](https://github.com/epfml/disco/assets/33122365/01721486-ffc2-4d2b-87e3-abfdd33260b3)

Currently:
![Screenshot 2024-07-01 at 15 05 40](https://github.com/epfml/disco/assets/33122365/aac9f966-8a3b-4b78-a92b-5ef1c9425117)